packages/iTwinUI-react/src/core/Table/columns/expanderColumn.tsx

```diff
@@ -62,7 +62,7 @@ export const ExpanderColumn = <T extends Record<string, unknown>>(
       } else {
```

Comment:
[Comment 1](https://github.com/iTwin/iTwinUI/pull/697#discussion_r918923600)

[Comment 2](https://github.com/iTwin/iTwinUI/pull/697#discussion_r921691827)

I didn't use the `table` prefix if the class name was a modifier instead of an element. `iui-slot` is a modifier for `iui-table-cell`. Another example of this is that `iui-sort` has the `table` prefix, but `iui-sorted` does not